docs/remote/faq.md

```diff
@@ -31,11 +31,11 @@ Some benefits of remote development include:
 
 Compared to using a network share or synchronizing files, VS Code Remote Development provides dramatically better performance along with better control over your development environment and tools.
 
-### How do the Remote Development extensions relate to Visual Studio Codespaces?
+### How do the Remote Development extensions relate to GitHub Codespaces?
 
-[Visual Studio Codespaces](https://aka.ms/vso) is a service that provides managed cloud-hosted development environments accessible from both VS Code and a new browser-based editor. The service also allows VS Code and the browser-based editor to access self-hosted environments (desktop or server) without requiring an SSH server or even a direct network route. You can read more about Visual Studio Codespaces in [its documentation](https://aka.ms/vso-docs).
+[GitHub Codespaces](https://github.com/features/codespaces) is a service that provides managed cloud-hosted development environments accessible from both VS Code and a new browser-based editor. The service also allows VS Code and the browser-based editor to access self-hosted environments (desktop or server) without requiring an SSH server or even a direct network route. You can read more about GitHub Codespaces in [its documentation](https://docs.github.com/en/github/developing-online-with-codespaces).
 
-While the Remote Development and Visual Studio Codespaces extensions share technology and features, the Remote Development extensions are released separately and can operate independently from Visual Studio Codespaces (and therefore do not require a Visual Studio Codespaces plan).
+While the Remote Development and Codespaces extensions share technology and features, the Remote Development extensions are released separately and can operate independently from GitHub Codespaces (and therefore do not require a GitHub Codespaces plan).
```

Comment:
We can remove the section in parentheses since there are no plans in GH Codespaces.

Reply:
Removed that section.